JSDemos/Demos/Charts/DrillDown/description.md

```diff
@@ -1,13 +1,9 @@
-This demo shows a drill-down chart that visualizes data on two hierarchical views. The main view displays the population breakdown by continent. When you click the bar, a detailed view reveals the most populated countries on the selected continent.
+When the data source has several hierarchical levels, you may want to enable users to navigate between them. The movement from the higher to the lower hierarchical level is called *"drilling down"*. The movement backwards is called *"drilling up"*.
```

Comment:
"When data source has". I prefer "if" to "when", unless you need to say "when" to indicate that something will definitely happen. "Has" is a weak verb. I would always try to replace with stronger subject-verb pair. 

The second and third sentences are somewhat parallel in structure and would work well in a list. 

I also wanted to highlight that you "can enable the feature", instead of "enable users to navigate". The latter sounds like some manual work (I will need to implement this on my own). But if we say "you can enable drill-down", it implies that this is a built-in functionality. 

```suggestion
If you bind a TreeMap to a data source with a hierarchical structure, you can enable drill-down and drill-up functionality:  

- *"Drill down"* means navigate to a more detailed view. 
- *"Drill up"* means navigate back up a level.
```